src/silx/app/view/CustomPlotSelectionWindow.py

```diff
@@ -226,22 +243,51 @@ def clearAll(self):
         self._xDataset = None
 
         self._yParent.removeRows(0, self._yParent.rowCount())
-        
-        self._reset()
+
+        self._setXFile()
+        self._addYFile()
 
         self._plot1D.clear()
+        self._plot1D.setXAxisLabel("X")
+        self._plot1D.resetZoom()
 
         self._updateYCurvesWithDefaultX()
 
     def _reset(self):
         self._xDataset = None
 
-        self._addXFile("")
-        self._addYFile("")
+        self._setXFile()
+        self._addYFile()
 
         self._plot1D.clear()
         self._updateYCurvesWithDefaultX()
 
+    def _createTooltipAttributes(self, url):
+        attributeDict = {}
+
+        if url is not None:
+            attributeDict["#Title"] = "HDF5 Dataset"
+            attributeDict["Name"] = self._getTextSplit(url.data_path())
+            attributeDict["Path"] = url.data_path()
+            attributeDict["File name"] = self._getTextSplit(url.file_path())
+        else:
+            pass
+        return attributeDict
+    
+    def _getDefautToolTip(self, url):
+        attrs = self._createTooltipAttributes(url)
+        title = attrs.pop("#Title", None)
+        if len(attrs) > 0:
+            tooltip = _utils.htmlFromDict(attrs, title=title)
+        else:
+            tooltip = ""
+
+        return tooltip
```

Comment:
As opposed to `Hdf5Item.py`, I think it would be simpler to make it in a single method

```suggestion    
    def _getToolTip(self, url):
        if url is None:
            return ""
        attrs = {
            "Name": self._getTextSplit(url.data_path()),
            "Path": url.data_path(),
            "File name": self._getTextSplit(url.file_path()),
        }
        return _utils.htmlFromDict(attrs, title="HDF5 Dataset")
```
(not tested)

and eventually renamed.